ingest/src/main/scala/demo/LandsatIngest.scala

```diff
@@ -129,40 +63,27 @@ object LandsatIngestMain extends Logging {
 
   def main(args: Array[String]): Unit = {
     logger.info(s"Arguments: ${args.toSeq}")
-    val config = Config.parse(args)
-    logger.info(s"Config: $config")
-
     implicit val sc = SparkUtils.createSparkContext("GeoTrellis Landsat Ingest", new SparkConf(true))
-    val images = Landsat8Query()
-      .withStartDate(config.startDate.toDateTimeAtStartOfDay)
-      .withEndDate(config.endDate.toDateTimeAtStartOfDay)
-      .withMaxCloudCoverage(config.maxCloudCoverage)
-      .intersects(config.bbox)
-      .collect()
-
-    logger.info(s"Found ${images.length} landsat images")
 
-    val bandsWanted = Array(
-      // Red, Green, Blue
-      "4", "3", "2",
-      // Near IR
-      "5",
-      "QA")
+    EtlConf(args).getEtlJobs.foreach { job =>
+      val etl = Etl(job, Etl.defaultModules :+ LandsatModule)
+      val inputPlugin = new TemporalMultibandLandsatInput()
```

Comment:
However, we need explicitly create input plugin class there, to have access to `var images` field, and not to download lc8 scenes twice